src/api/Contracts/Contracts.ts

```diff
@@ -0,0 +1,52 @@
+import { useQuery } from "@tanstack/react-query";
+import { subWebContext } from "src/api/SPWebContext";
+import { z } from "zod";
+import "@pnp/sp/items";
+
+const spContracts = z.array(
+  z.object({
+    Id: z.number().int().positive(),
+    Title: z.string().max(255, "Title must be 255 characters or less"),
+    ContractNumber: z.string(),
+    Contractor: z.object({
+      Id: z.number().int().positive(),
+      Title: z.string(),
+    }),
+    ContractorPOC: z
+      .object({
+        Id: z.number().int().positive(),
+        Title: z.string(),
+      })
+      .optional(),
+    DODAAC: z.string().length(6, "DODAAC must be 6 characters long"),
```

Comment:
Should this field in SharePoint be a lookup? So they can only select DODAACs they have added to the site?

Reply:
Per our discussion the other day, this isn't something worth implementing